Allowing people to get the authorization url instead of forcing the redirection

totophe opened the pull request:
This mechanism allow a better integration with frameworks allowing the developer handling the redirection rather than forcing it.

Commit:
Add Authorization URL function

Commit:
Add getAuthorizationUrl to the doc

Commit:
style fix

Commit:
Doc logic revisited

jabes commented:
I could use this on my current project, any word on when this will be implemented?

totophe commented:
I've just rewritten the part were it generate the url, moved it into a separate public function and "voilà" :-)

It should have been merged a long time ago, but it isn't :( You can use my branch, but it hasn't been updated in a while. I can probably do a pull from the galen repository if you want.

odino commented:
ping @galen

eschultz commented:
Sadly @galen passed away on 12/1/2013 due to non-hodgkins lymphoma. 😢

odino commented:
RIP :cry:

totophe commented:
Sad news :'( RIP
